### PR TITLE
Remove celery from gaia

### PR DESCRIPTION
**Pull request by danlipsa**

### CODE REVIEW

**Comment by danlipsa** [comment]
@aashish24 @matthewma7 Please review.

File: ansible/roles/romanesco-variables/defaults/main.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@
     romanesco_net_interface: eth0
 
     romanesco_main_app_name: romanesco
```
Comment (matthewma7) [inline ansible/roles/romanesco-variables/defaults/main.yml]: @aashish24 I am just curious what romanesco is, I think it's the old name for girder worker?

Comment (danlipsa) [reply ansible/roles/romanesco-variables/defaults/main.yml]: @aashish24 would know.

**Comment by jeffbaumes** [comment]
Yes - romanesco was renamed to girder_worker and has been reworked considerably since then.

**Comment by aashish24** [comment]
@matthewma7 we were using romanesco before and now since we decided (and are using) to use girder worker we can remove it.

**Comment by danlipsa** [comment]
@aashish24 @matthewma7 I think I don't have write access to this repo so I don't see a merge button.

**Comment by aashish24** [comment]
@danlipsa not sure what happened - I thought we added you..but here is the invite: https://github.com/OpenDataAnalytics/gaia/invitations

**Pull request merged**